### ISSUE TITLE
ERROR in chunk main [entry]

### ISSUE THREAD

**Issue by labnol** [fossamagna/gas-webpack-plugin#460]
Thanks for the good work.

I am getting an error after upgrading to v1.2.1.

```
ERROR in chunk main [entry]
090ffc44364d4dc2508e.js
/Users/labnol/node_modules/babel-loader/lib/index.js!/Users/labnol/node_modules/eslint-loader/dist/cjs.js??ref--4!/Users/labnol/src/index.js
```

If you can help, that would be awesome!

**Comment by fossamagna** [comment]
@labnol Thank you for your report.
Can I have the source codes(repository) to reproduce this error.

**Comment by labnol** [comment]
I've added you to the labnol/dummy repository.

You can do `npm run build` to reproduce the error.

**Comment by fossamagna** [comment]
@labnol Thank you. I could reproduce the error on `labnol/dummy` repository.

**Comment by fossamagna** [comment]
The error is occurring while trying to parse the ES6 syntax. It looks like you need to fix the [gas-entry-generator](https://github.com/fossamagna/gas-entry-generator).

**Comment by fossamagna** [comment]
@labnol I published new version (v1.2.2).
Please try it.

**Comment by labnol** [comment]
@fossamagna Thank you! Thank you!
